am-lib/src/main/java/uk/gov/hmcts/reform/amlib/AccessManagementService.java

```diff
@@ -17,4 +19,12 @@ public void createResourceAccess(String resourceId, String accessorId) {
         jdbi.useExtension(AccessManagementRepository.class,
             dao -> dao.createAccessManagementRecord(resourceId, accessorId));
     }
+
+    public List<String> checkAccess(UserDetails userDetails, String resourceId) {
```

Comment:
documentation needed

Reply:
I wonder if we actually want to use UserDetails here.